Makefile

```diff
@@ -195,18 +196,18 @@ test: all
 	$(MAKE) build-addons
 	$(MAKE) build-addons-napi
 	$(MAKE) cctest
-	$(PYTHON) tools/test.py --mode=release -J \
+	$(TEST_CI) \
 		doctool inspector known_issues message pseudo-tty parallel sequential $(CI_NATIVE_SUITES)
 	$(MAKE) lint
 
 test-parallel: all
-	$(PYTHON) tools/test.py --mode=release parallel -J
+	$(TEST_CI) parallel
 
 test-valgrind: all
 	$(PYTHON) tools/test.py --mode=release --valgrind sequential parallel message
 
 test-check-deopts: all
-	$(PYTHON) tools/test.py --mode=release --check-deopts parallel sequential -J
+	$(TEST_CI) --check-deopts parallel sequential
```

Comment:
Standardizing the calls to `python tools\test.py`
I admit that I'm not sure of the benefit of the changes in `Makefile`, especially if there is risk of breakage